### PR TITLE
fix: cpu request & limits were inverted in default benchmark

### PR DESCRIPTION
**Pull request by entangled90**
cpu requests & limits were inverted by mistake

### CODE REVIEW

File: charts/zeebe-benchmark/test/golden/c8-zeebe-statefulset.golden.yaml [hunk 1]
```diff
@@ -174,10 +174,10 @@ spec:
             timeoutSeconds: 1
           resources:
             limits:
-              cpu: 1350m
```
Comment (Zelldon) [inline charts/zeebe-benchmark/test/golden/c8-zeebe-statefulset.golden.yaml:177]: Interesting, did it fail?

**Review by Zelldon — APPROVED** [review]

File: charts/zeebe-benchmark/test/golden/c8-zeebe-statefulset.golden.yaml (hunk 1 above)
Comment (npepinpe) [reply charts/zeebe-benchmark/test/golden/c8-zeebe-statefulset.golden.yaml:177]: Yes :)

Comment (npepinpe) [reply charts/zeebe-benchmark/test/golden/c8-zeebe-statefulset.golden.yaml:177]:
```
49s (x17 over 63s)      Warning   FailedCreate             StatefulSet/medic-y-2024-cw-45-0a3701a8-benchmark-zeebe                     create Pod medic-y-2024-cw-45-0a3701a8-benchmark-zeebe-2 in StatefulSet medic-y-2024-cw-45-0a3701a8-benchmark-zeebe failed error: Pod "medic-y-2024-cw-45-0a3701a8-benchmark-zeebe-2" is invalid: spec.containers[0].resources.requests: Invalid value: "1700m": must be less than or equal to cpu limit of 1350m
```

**Review by npepinpe — APPROVED** [review]

File: charts/zeebe-benchmark/test/golden/c8-zeebe-statefulset.golden.yaml (hunk 1 above)
Comment (Zelldon) [reply charts/zeebe-benchmark/test/golden/c8-zeebe-statefulset.golden.yaml:177]: I guess we should have an integration test

**Pull request merged**